common/server_protocolhandler.cpp

```diff
@@ -60,27 +60,27 @@ void Server_ProtocolHandler::prepareDestroy()
     while (gameIterator.hasNext()) {
         gameIterator.next();
 
-        Server_Room *r = server->getRooms().value(gameIterator.value().first);
-        if (!r)
+        Server_Room *room = server->getRooms().value(gameIterator.value().first);
+        if (!room)
             continue;
-        r->gamesLock.lockForRead();
-        Server_Game *g = r->getGames().value(gameIterator.key());
-        if (!g) {
-            r->gamesLock.unlock();
+        room->gamesLock.lockForRead();
+        Server_Game *game = room->getGames().value(gameIterator.key());
+        if (!game) {
+            room->gamesLock.unlock();
             continue;
         }
-        g->gameMutex.lock();
-        Server_Player *p = g->getPlayers().value(gameIterator.value().second);
+        game->gameMutex.lock();
+        Server_Player *p = game->getPlayers().value(gameIterator.value().second);
         if (!p) {
```

Comment:
Is there a reason you left `p` unchanged?

Reply:
macros work very well for changes like this but don't have that kind of initiative